src/org/zaproxy/zap/extension/invoke/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Invoke Applications</name>
-	<version>8</version>
+	<version>9.0.0</version>
```

Comment:
- Do we want to use semver here?
 - Changes element needs to be updated (remove old change and add short description of the new parameters).

Reply:
I just guessed for this field, if you would rather we just increment it to `9`?

Reply:
That's the way it is normally handled

Reply:
I think so, we are still using that version scheme in all other add-ons (not sure we need semver for this one, yet :)